run_dir/static/js/data_del_plot.js

```diff
@@ -68,9 +68,18 @@ function make_plot(key, name, view_type, filter_inst_type){
             }
         },
         tooltip: {
+            shared: true,
             useHTML: true,
             headerFormat: '<span style="color:{point.color}">\u25CF</span><small>{point.key}</small><br />',
-            pointFormat: '{series.name} : <b>{point.y}</b>'
+            pointFormatter: function () {
```

Comment:
Slick! 🚀

Reply:
Thank you! I fixed the merge conflict as well....I think :-)